common.js

```diff
@@ -28,6 +28,11 @@ const defaultOptions = {
     everywhere: false,
   },
   thumbnailMode: 'hidden',
+  workMode: {
```

Comment:
should: call this `scheduledBlocking` or `timeBasedBlocking`, or similar. On first glance I don't think I'd know what `workMode` means.

Reply:
must: Update the typedef on lines 9-18 to match (GitHub won't let me comment there). This ensures intellisense works as expected, and in future if we convert to TypeScript / run linters they'll pass.

You probably will want to merge in the master branch first, because I think there may be a few merge conflicts otherwise.

Reply:
consider: store this as an array of enabled times. This would allow users to set multiple periods where the extension is enabled, for example if they have a break in their work day or similar.

We don't actually have to accept more than one to start with, but I think it'd be good to design this so that if we do want to accept more than one in future people don't lose their settings. The options page would either:
- have an input as just a text field, where people can enter comma-separated ranges - format could be like LeechBlock (e.g. `0900-1700,1800-1900`) or [ISO8601-like](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) format (e.g. `09:00/17:00,18:00/19:00`)
- have pairs of time inputs, and users could click a `+` icon to add more ranges